projects/knora-ui/package.json

```diff
@@ -1,6 +1,11 @@
 {
-  "name": "knora-ui",
+  "name": "@knora/ui",
   "version": "1.0.0-alpha",
```

Comment:
Should we call the first version `1.0.0-alpha.1`? See https://docs.npmjs.com/misc/semver#prerelease-tags

Reply:
The first version should be 1.0.0-PR1 as mentioned in design documentation and on youtrack. I will update the version number in this PR.

Reply:
PR = Prerelease? I think we could also call it RC (release candidate). @subotic?

Reply:
ah yes, RC is much better. What do you think @subotic? Can we change from PR to rc?